app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java

```diff
@@ -111,18 +113,20 @@ private void createEventHeader(LinearLayout rootView) {
         // initialize all buttons
         Button link = headerView.findViewById(R.id.button_link);
         Button ticket = headerView.findViewById(R.id.button_ticket);
+        Button export_calendar = headerView.findViewById(R.id.button_export_eventcalendar);
```

Comment:
Use the java naming convention for variables: exportCalendar

Reply:
fixed it . Thx